src/containers/walletConnect/selectAppModal/index.tsx

```diff
@@ -38,13 +39,34 @@ export const AllowListDApps: AllowListDApp[] = [
   },
 ];
 
+export const enableConnectAnyDApp =
+  featureFlags.getValue('VITE_FEATURE_FLAG_CONNECT_ANY_DAPP') === 'true';
+
+if (import.meta.env.DEV || enableConnectAnyDApp) {
```

Comment:
If we add the `VITE_FEATURE_FLAG_CONNECT_ANY_DAPP=true` feature flag to the `.env.development` environment file as well, we can just check for the `enableConnectAnyDApp` variable here and everywhere we need it as the `.env.development` environment var files should be picked up when developing locally
```suggestion
if (enableConnectAnyDApp) {
```